### ISSUE TITLE
Converting HTTP Response to JSON misses some quotation marks

### ISSUE THREAD

**Issue by DirtyDevWork** [DirtyDevWork/homebridge-daikin#8]
An example output from convertDaikinToJSON():
`{"ret":"OK,pow":"0,mode":"4,adv":",stemp":"21.0,shum":"0,dt1":"25.0,dt2":"M,dt3":"25.0,dt4":"21.0,dt5":"21.0,dt7":"25.0,dh1":"AUTO,dh2":"50,dh3":"0,dh4":"0,dh5":"0,dh7":"AUTO,dhh":"50,b_mode":"4,b_stemp":"21.0,b_shum":"0,alert":"255"}`

What should be output:
`{"ret":"OK","pow":"0","mode":"4","adv":"","stemp":"21.0","shum":"0","dt1":"25.0","dt2":"M","dt3":"25.0","dt4":"21.0","dt5":"21.0","dt7":"25.0","dh1":"AUTO","dh2":"50","dh3":"0","dh4":"0","dh5":"0","dh7":"AUTO","dhh":"50","b_mode":"4","b_stemp":"21.0","b_shum":"0","alert":"255"}`

Basically, need to work out why quotation marks are not being put around the commas

**Comment by DirtyDevWork** [comment]
Still not resolved, bear with

**Comment by DirtyDevWork** [comment]
Still didn't work, added backslashes in but did not resolve comma issue

**Comment by DirtyDevWork** [comment]
Issue resolved, however doing so leave the plugin only reading info from the Daikin system, but unable to push changes out to it.
